test/factories/budgets_factory.rb

```diff
@@ -93,9 +93,7 @@ def teardown
   private
 
   def bulk(params = {})
-    result = client.bulk(params)
-    sleep 1 # wait for search index to be ready
-    result
+    client.bulk(params.merge(refresh: true))
```

Comment:
When I originally did this I tried this option but for some reason it was not working, so I used the sleep. Tests are passing so good to me.